controllers/accountpool/accountpool_controller.go

```diff
@@ -106,6 +114,61 @@ func (r *AccountPoolReconciler) Reconcile(ctx context.Context, request ctrl.Requ
 	return reconcile.Result{}, nil
 }
 
+func (r *AccountPoolReconciler) handleServiceQuotas(reqLogger logr.Logger, account *awsv1alpha1.Account) error {
+	reqLogger.Info("Loading Service Quotas")
+
+	cm, err := utils.GetOperatorConfigMap(r.Client)
+	if err != nil {
+		reqLogger.Error(err, "failed retrieving configmap")
+		return err
+	}
+
+	accountpoolString, found := cm.Data["accountpool"]
+	if !found {
+		reqLogger.Error(fixtures.NotFound, "failed getting accountpool data from configmap")
+		return fixtures.NotFound
+	}
+
+	type Servicequotas map[string]string
```

Comment:
Could this reuse the `ServiceQuotas` type from account_types?

Reply:
This one is different because the configmap holds a nested YAML for the servicequota definition we want to load. 
Because of the nested YAML we can't load it in one step in go, but instead have to load a string first and then convert it to the actual type later in this function:

This is a configmap example we use to test:
```yaml
accountpool: "\nzero-size-accountpool: \n  default: true\nhs-zero-size-accountpool:\n
    \ servicequotas:\n    default:\n      L-1216C47A: '10'\n      L-0263D0A3: '6'\n
    \   us-east-1:\n      L-1216C47A: '11'"
```

And the value of the accountpool must first be loaded as string as it's still yaml, after the first Unmarshal ...